### PR TITLE
Remove alchy

### PR DESCRIPTION
**Pull request by seallard**
## Description
Replacing alchy with sqlalchemy in Housekeeper.

### Changed
-  Replaced `commit` with `session.commit`
- Replaced `add_commit` with `session.add` followed by `session.commit`
- Replaced `delete` with `session.delete`
- Replaced all `alchy` imports with `sqlalchemy`
- The Store is now instantiated with a session, which is sent up to all the handlers.
- Remove alchy in `requirements.txt`

### How to prepare for test

- [ ] ssh to hasta (depending on type of change)
- [ ] install on stage:
```bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_main -t housekeeper -b YOURBRANCH```

### How to test
- [ ] login to ...
- [ ] do ...

### Expected test outcome
- [ ] check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review
- [ ] code approved by
- [ ] tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

This [version](https://semver.org/) is a:
- [ ] **MINOR** - when you add functionality in a backwards compatible manner

### CODE REVIEW

File: housekeeper/store/api/base.py [hunk 1]
```diff
@@ -1,30 +1,32 @@
...
     """This is a base class holding different models"""
 
-    Bundle: Type[ModelBase] = Bundle
-    Version: Type[ModelBase] = Version
-    File: Type[ModelBase] = File
-    Tag: Type[ModelBase] = Tag
+    Bundle: Type[Model] = Bundle
+    Version: Type[Model] = Version
+    File: Type[Model] = File
+    Tag: Type[Model] = Tag
```
Comment (Vince-janv) [inline housekeeper/store/api/base.py:14]: Do we need this anymore? I'm not sure why it was there in the first place?

**Review by Vince-janv — APPROVED** [review]
Nice! 👌

File: housekeeper/store/api/base.py (hunk 1 above)
Comment (seallard) [reply housekeeper/store/api/base.py:14]: Hm, me neither! Good catch. But the tests fail when I remove them, seems like it requires some more refactoring! I'll do it in a separate PR

File: housekeeper/store/api/add.py [hunk 2]
```diff
@@ -15,9 +16,8 @@
 class AddHandler(BaseHandler):
     """Handles adding things to the store"""
 
-    def __init__(self):
-        super().__init__()
-        AddHandler.version = FindHandler.version
```
[the remark this answers is not in the document]
Comment (seallard) [reply housekeeper/store/api/add.py:20]: AddHandler.version is not used anywhere, and the FindHandler also does not define any version method. But good catch! There are methods missing from the FindHandler. I don't really like this way of accessing methods from the other handlers, we should find a better solution.

[the remark this answers is not in the document]
Comment (seallard) [reply housekeeper/store/api/add.py:20]: https://github.com/Clinical-Genomics/housekeeper/issues/113

[the remark this answers is not in the document]
Comment (seallard) [reply housekeeper/store/api/add.py:20]: Added the missing find method: https://github.com/Clinical-Genomics/housekeeper/pull/112/commits/72eae4b73953fa8adf2a0b96e9f19571215b1f12

**Comment by seallard** [comment]
Deployed to stage, can retrieve tags for instance with `housekeeper get tag`

**Pull request merged**